CMakeLists.txt

```diff
@@ -233,6 +233,34 @@ else ()
   message(STATUS "Using agora raw logger")
 endif()
 
+set(USE_CSV_LOGGER False CACHE STRING "USE_CSV_LOGGER defaulting to 'False'")
+message(STATUS "USE_CSV_LOGGER: ${USE_CSV_LOGGER}")
+if(USE_CSV_LOGGER AND USE_SPDLOG)
+  if(LOG_LEVEL STREQUAL "error")
```

Comment:
Let's remove the LEVEL selection from the CSV log since it is not implemented yet

Reply:
done.